libnavigation-base/src/main/java/com/mapbox/navigation/base/routerefresh/RouteRefreshAdapterProvider.kt

```diff
@@ -0,0 +1,11 @@
+package com.mapbox.navigation.base.routerefresh
+
+object RouteRefreshAdapterProvider {
```

Comment:
name suggestion: RouteOptionsProvider instead of RouteRefreshAdapterProvider

Reply:
maybe `RoutingOptionsProvider` more for _process_, not for _static_ route?